src/components/forms/controls/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,46 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { classNames as cx, type ComponentProps } from '../../../../util/componentUtil.ts';
+
+import cl from './Textarea.module.scss';
+
+
+export { cl as TextareaClassNames };
+
+export type TextareaProps = Omit<ComponentProps<'textarea'>, 'type'> & {
+  /** Whether this component should be unstyled. */
+  unstyled?: undefined | boolean,
+
+  /** 
+   * Whether the textarea should resize automatically, with `field-sizing: content`.
+   * Note that browser support is still somewhat limited:
+   * https://developer.mozilla.org/en-US/docs/Web/CSS/field-sizing
+   */
+  automaticResize?: undefined | boolean,
```

Comment:
I think this should have the option to only resize vertically or horizontally or both. Usually we'd only want the text area to grow vertically, not horizontally.

Reply:
I am simply forwarding the CSS `field-sizing: content` which does both. The way to resize only vertically would be by setting a fixed `width` to it. Nevertheless I created a separate story exploring that option.